.github/workflows/conventional-pr-title.yml

```diff
@@ -0,0 +1,17 @@
+name: Conventional PR Title
+on:
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+      - edited
+      - synchronize
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    permissions:
+      statuses: write
+    steps:
+      - uses: aslafy-z/conventional-pr-title-action@v3
```

Comment:
This checks that the PR metadata conforms to the conventions, not the commits in the branch, correct?

I think this granularity of enforcement will be good, if we keep the squash-and-merge method of merging PRs.

Reply:
> not the commits in the branch, correct?

correct ✅